cursorless-talon/src/cursorless_global.talon

```diff
@@ -0,0 +1,3 @@
+{user.cursorless_homophone} (reference | ref | cheatsheet | cheat sheet): user.cursorless_cheat_sheet_show_html()
```

Comment:
Make all these active globally, though the cheatsheet just shows a helpful message, per above